doc/classes/Image.xml

```diff
@@ -472,11 +472,22 @@
 			</argument>
 			<description>
 				Sets the [Color] of the pixel at [code](x, y)[/code]. Example:
-				[codeblock]
+				[codeblocks]
+				[gdscript]
+				var img_width = 10
+				var img_height = 5
 				var img = Image.new()
 				img.create(img_width, img_height, false, Image.FORMAT_RGBA8)
-				img.set_pixel(x, y, color)
-				[/codeblock]
+				img.set_pixel(2, 2, Color.red)
```

Comment:
Changing from `x, y` to `2, 2` to make it a valid code example may or may not be a good idea, maybe we should instead have `var x = 2` etc (CC @akien-mga).

At the very least now it's inconsistent with the comment above the code example, so this isn't good as-is.

P.S. @akien-mga We need to replace `set_pixel` and `set_pixelv` with a version that uses `Vector2i` for 4.0.

Reply:
We should rename the parameter name from dst to destination or something better. I could do that in the PR too

Reply:
Maybe `Sets the [Color] of the pixel at the position supplied by the x and y parameters.` Not nice, but perhaps something in that direction?

Reply:
I opened #43627 for refactoring this. Since there isn't a point to mention x and y individually there, I made the code example just have the vector created inline with the method call.